apps/console/src/features/identity-verification-providers/components/identity-verification-provider-edit.tsx

```diff
@@ -0,0 +1,213 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { IdentifiableComponentInterface } from "@wso2is/core/models";
+import {
+    ResourceTab,
+    ResourceTabPaneInterface
+} from "@wso2is/react-components";
+import React, {
+    FunctionComponent,
+    ReactElement, useEffect,
+    useState
+} from "react";
+import { TabProps } from "semantic-ui-react";
+import {
+    AttributeSettings,
+    GeneralSettings
+} from "./settings";
+import { ConfigurationSettings } from "./settings/configuration-settings";
+import { IdentityVerificationProviderConstants } from "../constants/identity-verification-provider-constants";
+import {
+    IDVPClaimMappingInterface, IDVPClaimsInterface, IdentityVerificationProviderInterface
+} from "../models";
+import { UIMetaDataForIDVP } from "../models/ui-metadata";
+import { getContentLoader } from "../utils";
+
+/**
+ * Prop types for the identity verification provider edit component.
+ */
+interface EditIdentityVerificationProviderPropsInterface extends IdentifiableComponentInterface {
+    /**
+     * Editing idvp.
+     */
+    identityVerificationProvider: IdentityVerificationProviderInterface;
+
+    /**
+     * Is the data still loading.
+     */
+    isLoading?: boolean;
+    /**
+     * Callback to be triggered after deleting the identity verification provider.
+     */
+    onDelete: () => void;
+    /**
+     * Callback to update the identity verification provider details.
+     */
+    onUpdate: () => void;
+    /**
+     * Specifies if the component should only be read-only.
+     */
+    isReadOnly: boolean;
+    /**
+     * Specifies if it is needed to redirect to a specific tabindex
+     */
+
+    uiMetaData: UIMetaDataForIDVP;
+
+    isAutomaticTabRedirectionEnabled?: boolean;
```

Comment:
Let's add a comment here to explain the purpose of `isAutomaticTabRedirectionEnabled`

Reply:
Resolve by 67f70ee693d83b7e537215111173b2a69b810b79